bin/test-exercise.sh

```diff
@@ -8,6 +8,9 @@ if [ -z "$exercise_path" ]; then
     exit 1
 fi
 
+# bust the cache
+touch "$exercise_path/src/lib.rs"
```

Comment:
What does this do?

Reply:
The whole `bin/test-exercise.sh` script is for local testing based on the idea that I have a directory full of completed exercises to send through the test runner; it's not on the critical path as part of the spec.

The point of this line is to bust Docker's caching. If I test, say, my implementation of `clock`, then Docker caches that implementation. If I then want to re-run the test without changing the `clock` implementation, Docker will spit out the same output as before without actually re-running the test suite. 

The `touch` command creates a file if it doesn't exist, but it also ensures that the file's modification timestamp is current without otherwise modifying it if it does. By adding this line, we bypass Docker's cache, because we guarantee that the file's modification is more recent than the most recent Docker run.